src/assets/i18n/de.json

```diff
@@ -1,59 +1,95 @@
-  "anms.about.get-notified": "Werde über neue Versionen informiert",
-  "anms.about.get-notified.follow": "Folge",
-  "anms.about.get-notified.description": "ein Twitterbot, der dich über die neusten Versionen deiner Lieblingsframeworks & -bibliotheken informiert!",
-  "anms.about.contributors": "Beitragende",
-  "anms.about.contributors.description1": "Möchtest du auch zu einem Open Source Projekt mit Angular beitragen?",
-  "anms.about.contributors.description2": "Hinterlasse deine Spuren und werde Teil eines wachsenden Teams!",
-  "anms.about.contributors.description3": "Werfe einen ersten Blick auf",
-  "anms.about.contributors.description4": " und ",
-  "anms.about.contributors.issues": "offene Probleme",
-  "anms.about.contributors.guide": "die Anleitung für Beitragende",
-  "anms.features.title": "Features",
-  "anms.features.documentation": "Doku",
-  "anms.features.guide": "Anleitung",
-  "anms.features.angular": "Modernes und leistungsfähiges Framework",
-  "anms.features.angular-material": "Material Design-Komponente",
-  "anms.features.angular-cli": "Eine Befehlszeilenschnittstelle für Angular",
-  "anms.features.ngrx": "Einweg-Datenfluss unterstützt von RxJS Observables",
-  "anms.features.ngxtranslate": "Eine Internationalisierungsbibliothek für Angular",
-  "anms.features.rxjs": "Reaktive Programmierung mit Async-Sammlungen unter Verwendung von Observablen",
-  "anms.features.bootstrap": "Responsives Layout von der weltbekannten UI-Bibliothek",
-  "anms.features.typescript": "Überlegene Entwickler Erfahrung, Code-Vervollständigung, Refaktorierung und weniger Bugs",
-  "anms.features.fontawesome": "Einfach zu bedienende Vektor-Icons und soziale Logos",
-  "anms.features.themes": "Flexible Theming-Unterstützung für bereitgestellte und benutzerdefinierte Komponenten.",
-  "anms.features.themes.title": "Themen",
-  "anms.features.lazyloading": "Schnellere Startzeit mit Lazy Loaded Feature-Modulen",
-  "anms.features.lazyloading.title": "Lazy Loading",
-  "anms.settings.title": "Einstellungen",
-  "anms.settings.general": "Haupteinstellungen",
-  "anms.settings.general.placeholder": "Sprache",
-  "anms.settings.general.language.en": "Englisch",
-  "anms.settings.general.language.sk": "Slovakisch",
-  "anms.settings.general.language.de": "Deutsch",
-  "anms.settings.general.language.fr": "Französisch",
-  "anms.settings.general.language.pt-br": "Portugiesisch",
-  "anms.settings.themes": "Thema",
-  "anms.settings.themes.placeholder": "Farbthema",
-  "anms.settings.themes.blue": "Blau",
-  "anms.settings.themes.light": "Hell",
-  "anms.settings.themes.nature": "Natur",
-  "anms.settings.themes.dark": "Dunkel",
-  "anms.settings.themes.night-mode": "Auto Nachtmodus (von 21:00 bis 7:00)",
-  "anms.settings.animations": "Animationen",
-  "anms.settings.animations.page": "Navigation ganzer Seitenübergang",
-  "anms.settings.animations.elements": "Navigation Seitenelemente nach oben schieben"
+	"anms": {
```

Comment:
I am against nested properties for translations, in components we use them as for example `anms.features.lazyloading.description`, we want them in same format also in the translation files for the convenience...